cognite_toolkit/_cdf_tk/data_classes/_config_yaml.py

```diff
@@ -58,44 +58,35 @@
 class Environment:
     name: str = "dev"
     project: str = field(default_factory=lambda: os.environ.get("CDF_PROJECT", "UNKNOWN"))
-    build_type: EnvType = DEFAULT_ENV  # type: ignore[assignment]
+    validation_type: str = "dev"
```

Comment:
Keeping the default to `dev` as setting it to `prod` would be a breaking change.